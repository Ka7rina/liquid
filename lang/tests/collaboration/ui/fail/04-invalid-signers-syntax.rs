use liquid_lang as liquid;

#[liquid::collaboration]
mod noop {
    #[liquid(contract)]
    pub struct Noop {
        #[liquid(signers = "$[..][..][1,2,3]@(?true)[..][..]")]
        addr: u8,
    }
}


fn main() {}
