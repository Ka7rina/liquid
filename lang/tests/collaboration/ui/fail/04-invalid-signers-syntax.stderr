error: around "3]@(?"(at position 16), expected one of `.`, `[`, `(?`, `END`, `)`, `&&`, `||`, `<`, `<=`, `>`, `>=`, `==` or `!=`, but found `@`
 --> $DIR/04-invalid-signers-syntax.rs:7:28
  |
7 |         #[liquid(signers = "$[..][..][1,2,3]@(?true)[..][..]")]
  |                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
